agent/asteroid_agent.py

```diff
@@ -59,15 +59,17 @@ def process(self, message: m.Message) -> None:
         Args:
             message: message containing the asset to scan.
         """
+        logger.info("Preparing targets ...")
         targets = targets_preparer.prepare_targets(message)
         with futures.ThreadPoolExecutor() as executor:
             targets_checks = [
                 executor.submit(_check_target, exploit, target)
                 for target in targets
                 for exploit in self.exploits
             ]
-
+            logger.info("Scanning targets `%s`.", targets)
             for target_vulnz in futures.as_completed(targets_checks):
+                logger.info("Found %d vulnerabilities", len(target_vulnz.result()))
```

Comment:
Can this message be more specific to tell which target the vulnerabilities were found in?